spec/services/decorators/v4/disposable_income_result_decorator_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+module Decorators
+  module V3
```

Comment:
File and module name mismatch, V4 but module still v3

Reply:
👍